ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts

```diff
@@ -87,6 +87,8 @@ private transformationPaths: MigrationPath[] = [
     isInWizard: boolean;
     action: Action;
 
+    static DEFAULT_MIGRATION_PATH: MigrationPath = <MigrationPath>{ id: 101 };
```

Comment:
so you prepopulated it in component, crap, I forget to think about it this way. OK, as I have transformationPaths there, I will use it directly. Thanks for pointing to that.